src/ArToolkitContext.ts

```diff
@@ -40,7 +40,7 @@ export class ArToolkitContext implements IArToolkitContext {
             matrixCodeType: '3x3',
 
             // url of the camera parameters
-            cameraParametersUrl: ArToolkitContext.baseURL + '../data/camera_para.dat',
+            cameraParametersUrl: ArToolkitContext.baseURL + '../../data/data/camera_para.dat',
```

Comment:
ArToolkitContext.baseURL seems have no effect, infact the ArToolkitContext fails to load the default file.